src/utils/pause/frames/displayName.js

```diff
@@ -91,7 +92,9 @@ export function formatDisplayName(
     displayName = mapDisplayNames(frame, library);
   }
 
-  return simplifyDisplayName(displayName) || L10N.getStr("anonymousFunction");
```

Comment:
perhaps we should not have `L10N` in the util. I'd suggest doing L10N in the component that calls the util. Given that we have many call sites, we might want to have a shared functional component.